.github/workflows/maven.yml

```diff
@@ -21,5 +21,7 @@ jobs:
       with:
         java-version: '11'
         distribution: 'adopt'
+    - name: Run Maven Install
+      run: mvn install
```

Comment:
This is a minor comment, but this will run the tests twice. If you change the below command from `package` to `install` it would do the same thing.

Reply:
okay!